docs/src/staking/README.md

```diff
@@ -0,0 +1,116 @@
+# Staking on Solana
+
+Staking your SOL tokens on Solana is the best way you can help secure the world's
+highest-performing blockchain network, and
+[earn rewards](../implemented-proposals/staking-rewards.md) for doing so!
+
+Solana is a Delegated Proof-of-Stake system (DPoS), which means that anyone who
+holds SOL tokens can choose to delegate some of their SOL to one or more
+validators, who process transactions and run the network.
+
+Delegating stake is a shared-risk shared-reward financial model that can provide
+steady returns to tokens held for a long period, similar to a traditional
+fixed-income investment.  This is achieved by aligning the financial incentives
+of the token-holders (delegators) and the validators to whom they delegate.
+
+The more stake a validator has delegated to them, the more often this validator
+is chosen to write new transactions to the ledger.  The more transactions
+the validator writes, the more rewards they and their delegators earn.
+Validators who configure their systems to be able to process more transactions
+at a time not only earn proportionally more rewards for doing so, they also
+keep the network running as fast and as smoothly as possible.
+
+Validators incur costs by running and maintaining their systems, and this is
+passed on to delegators in the form of a fee collected as a percentage of
+rewards earned.  This fee is known as a *commission*. As validators earn more
+rewards the more stake is delegated to them, they may compete with one another
+to offer the lowest commission for their services, in order to attract more
+delegated stake.
+
+There is a risk of loss of tokens when staking, through a process known as
+*slashing*.  Slashing is *NOT* presently implemented on Solana's Mainnet Beta
```

Comment:
Instead of "implemented", you can just say "enabled". The tech isn't the hard part, so don't want to indicate that it's some big missing piece. It's more about getting the community aligned on the ruleset.